test_conformance/math_brute_force/Utility.h

```diff
@@ -29,6 +29,7 @@
 #include "harness/testHarness.h"
 #include "harness/ThreadPool.h"
 #include "harness/conversions.h"
+#include "FunctionList.h"
```

Comment:
I'm not going to lose sleep over this, but it does feel a little awkward to include this file here.  Consider forward declaring `Func` instead?

Reply:
The implementation still needs to be able to de-reference `Func`.

Reply:
I could also just move it into 'FunctionList.cpp' which might be a more logical place anyway?

Reply:
Right, the implementation needs to dereference `Func`, but that just means that `Utility.cpp` needs to include `FunctionList.h`, not `Utility.h`.

I thought about moving it to `FunctionList.cpp` also, but because the implementation accesses `gIsEmbedded`, it'd mean that `FunctionList.cpp` would need to include `Utility.h`, which also feels a bit awkward.

I'm honestly fine leaving this as-is.  It may be that it's awkward regardless, and I agree that having a utility function is a nice improvement.